configure.ac

```diff
@@ -191,7 +191,7 @@ AS_IF([test "x$with_crypto" != "xno"],
       AS_IF([$PKG_CONFIG --atleast-version=2.0.3 libcryptsetup],
             [AC_DEFINE([LIBCRYPTSETUP_2])], [])
       AS_IF([$PKG_CONFIG --atleast-version=2.3.0 libcryptsetup],
-            [AC_DEFINE([LIBCRYPTSETUP_BITLK])], [])
+            [AC_DEFINE([LIBCRYPTSETUP_23])], [])
```

Comment:
Why not define both?

Reply:
I didn't want to add another variable and we already have `LIBCRYPTSETUP_2` and `LIBCRYPTSETUP_24` so `LIBCRYPTSETUP_23` looked like a better choice for both integrity and bitlk.